### DISCUSSION TITLE
Адаптация под вертикальный/мобильный экран

### DISCUSSION THREAD

**Discussion by Jaymz665** [anilibria/anilibria-winmaclinux#115] [Идеи]
Я конечно подозреваю, что я единственный  м̶#̶д̶а̶к̶ человек, который юзает мобильный линукс из тех, кто смотрит anilibria, но всё же. Единственный вариант просомтра приложения более или менее адекватно видится с жутким скейлом на phosh, про plasma mobile я и говрить не буду, там скейла нет. Учитывая что сайт так же не имеет мобильной адаптации, то единсвенный вариант это телега, но это лишний танец с бубном, т.к. там тоже не все "щикарно". Быть может есть возможность сделать адаптацию приложения, чтобы его можно было адекватно открыть на экране смартфона? В принципе даже не нужно менять интерфейс достаточно оставить один ряд предпросмотра и сделать чуть уже

**Comment by trueromanus** [comment]
Здравствуйте, подскажите пожалуйста что такое "достаточно оставить один ряд предпросмотра"? Мобильный линукс работает как и андроиды т.е. фиксированный размер окна который меняется только после переворота с альбома на портрет?

**Reply by Jaymz665** [reply]
Разрешение экрана смартфона 1080x2246 при применении скейла в 300% на phosh приложения открываются в разрешении 360X749, всё что не может подстроиться вылезает за рамки. На phosh имеется подгонка, но интерфейс становится мелким и неюзабельным. Я сейчас использую Plasma-mobile там с скейлом немного легче 240% вполне удобно для пользования и минимальное разрешение приложений становится 450x936, но к сожалению приложение Anilibria не влезает в эти рамки ни в вертикальном положении ни в горизонтальном положении экрана. Вот я и хотел узнать, можно ли адаптировать приложение урезав пустые поля. Что косается колонок, тут моя ошибка не обратил внимания, сокращение до 1 столбца всё же есть. 

![изображение](https://user-images.githubusercontent.com/54475558/209552182-232981dc-512d-4da5-b517-695e020a8326.png)

![изображение](https://user-images.githubusercontent.com/54475558/209552727-74b30b13-7f50-4b4f-b4e5-6a36cd21d2cf.png)

**Reply by trueromanus** [reply]
Там еще есть компактный режим но скорее всего он не особо улучшит ситуацию. Можно сделать верстку под мобильные устройства но это дело не быстрое, запланирую эту работу на следующий год. Спасибо за информацию!

**Reply by Jaymz665** [reply]
Большое спасибо, да компактный режим есть, но он наоборот прибавляет полей.
